idaes/apps/caprese/tests/test_controller.py

```diff
@@ -325,5 +325,5 @@ def test_constrain_control_inputs_piecewise_constant(self):
             pwc_expr = controller.pwc_constraint[i,t].expr
             tn = time.next(t)
             inputs = controller.vectors.input
-            pred_expr = (inputs[i, tn] - inputs[i, t] == 0.)
+            pred_expr = (inputs[i, tn] == inputs[i, t])
```

Comment:
Interesting that this has changed. I was relying on undefined behavior and will need to be more careful in the future.